README.md

```diff
@@ -22,14 +26,33 @@ not to show a table listing all the found file extensions and their respective
 frequencies, so that it will only display the total number of files.
 
 By default, it will ignore files and directories that are supposed to be
-hidden (with names starting with '.', but you can add the `-a` or `--all` optional
+hidden, but you can add the `-a` or `--all` optional
 switch argument to make it count all files.
 
+Hidden files and directories
+
+Windows: files and directories for which FILE_ATTRIBUTE_HIDDEN is true
+
+Linux, Mac OS: those with names starting with "."(dot)
+
 This utility can also be used to search for files that have a certain file extension
 (using `-fe` or `--file-extension`) and, optionally, display a short preview (`-p`or 
 `--preview`) for text files. The size of the preview text sample can optionally be
 customized by using the `-ps` or `--preview-size` argument followed by an integer number.
 
+The list of file types for which preview is available can be viewed with the `-st` or `--supported-types` argument.
+The names of extensions are case sensitive. The results for `ini` and `INI` will be different.
+
+By default, the result of a search by certain file extension is a list with the full paths of the files found.
+If you only need the total number of files, use the `-nl` or `--no-list` argument.
+
+The program's operating indicator is printing processed file names in one line.
+File names are not displayed when searching for a particular extension, if there are no such files in the folder or if the files are hidden, and the argument `--all` is not specified.
+
+Feedback is available by default for counting files by extension(table and no-table),
+searching for files by extension(viewing mode no-list). Optional argument `-nf` or `--no-feedback` disables it.
```

Comment:
I would add a notice here explaining that for some long running tasks the option for hiding feedback could mean a small but still noticeable improvement in speed.